src/managers/XWaylandManager.cpp

```diff
@@ -9,6 +9,10 @@
 
 CHyprXWaylandManager::CHyprXWaylandManager() {
 #ifndef NO_XWAYLAND
+    static auto* const XWAYLAND_ENABLED = &g_pConfigManager->getConfigValuePtr("xwayland:enabled")->intValue;
```

Comment:
style nit: no_snake

Reply:
that's screaming snake case, which is even used above )

all capitals just doesn't exist. thats little offtopic for a future

Reply:
> all capitals just doesn't exist. thats little offtopic for a future

I'd like to contest that information

![image](https://github.com/hyprwm/Hyprland/assets/43317083/199117ac-dc20-4307-8b1d-a6c69b711adb)

Reply:
> > all capitals just doesn't exist. thats little offtopic for a future
> 
> I'd like to contest that information
> 
> ![image](https://user-images.githubusercontent.com/43317083/269082034-199117ac-dc20-4307-8b1d-a6c69b711adb.png)

thanks for showing me terrible code)